git-https-proxy/main.go

```diff
@@ -0,0 +1,190 @@
+package main
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/elazarl/goproxy"
+)
+
+func main() {
+	config := parseEnv()
+	proxyHandler := getProxyHandler(config)
+	proxyServer := http.Server{
+		Addr:    fmt.Sprintf(":%s", config.ProxyPort),
+		Handler: proxyHandler,
+	}
+	healthHandler := getHealthHandler(config)
+	healthServer := http.Server{
+		Addr:    fmt.Sprintf(":%s", config.HealthPort),
+		Handler: healthHandler,
+	}
+	go func() {
+		// Run the health server in the "background"
+		log.Printf("Health server active on port %s\n", config.HealthPort)
+		log.Fatalln(healthServer.ListenAndServe())
+	}()
+	log.Printf("Git proxy active on port %s\n", config.ProxyPort)
+	log.Printf("Repo Url: %v, anonymous session: %v\n", config.RepoUrl, config.AnonymousSession)
```

Comment:
```suggestion
	log.Println("Git proxy active on port", config.ProxyPort)
	log.Println("Repo Url:" config.RepoUrl, "anonymous session:", config.AnonymousSession)
```
as `%v` is anyways default formatting, or?